### DISCUSSION TITLE
"y" to yank not working

### DISCUSSION THREAD

**Discussion by intercepted16** [JetBrains/ideavim#937] [Q&A] [answered]
I tried to use "y" to yank after "Shift + V" but it doesn't seem to work. Could someone please help me? I don't have an `.ideavimrc` file.

**Comment by citizenmatt** [comment, marked as answer]
`y` is used to yank to a specific register, identified with a letter or other symbol, and addressed with the `"` symbol, such as `"ayw` to yank a word to the "a" register, and `"ap` to put the contents of the "a" register. If you don't specify a register, Vim (and also IdeaVim) will use the "unnamed" register, which you can also access with the quote symbol, e.g. `""yw` or `""p`.

If you want to yank to the system clipboard, you need to use the clipboard register, which is "*", so `"*yw` and `"*p`. You can also configure this to be used instead of the unnamed register, so `yw` and `p` would use the system clipboard. You can do this by typing `:set clipboard=unnamed` and hitting return. If you want this to always be set, add a `~/.ideavimrc` file and add `set clipboard=unnamed`.
